migration/v2.18.0-ck8s1-v2.19.x-ck8s1/migrate-to-containerd.yml

```diff
@@ -55,6 +55,18 @@
           ansible.builtin.debug:
             var: clusterymloutput
 
+        - name: Delete remaining docker files
```

Comment:
Do we want to delete them in this step? It would be nicer to do it after we've migrated all so that we can revert back if in the middle something goes wrong

Reply:
But at the same time if we just re-add docker it will probably set itself up again so maybe this is a non-issue